### ISSUE TITLE
white background after burner climax vulkan and opengl

### ISSUE THREAD

**Issue by mkultimate** [RPCS3/rpcs3#5065]
![image](https://user-images.githubusercontent.com/16367065/44889463-a3093c80-ac9b-11e8-89e5-3c5f19a3f404.png)

![image](https://user-images.githubusercontent.com/16367065/44889491-bddbb100-ac9b-11e8-83e5-752253c485cf.png)

log
[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/2338560/RPCS3.log.gz)

**Comment by AniLeo** [comment]
Likely regressed on #5038, can you confirm?

**Comment by mkultimate** [comment]
@AniLeo recently I just tried it and have that problem

**Comment by Emulator-Team-2** [comment]
@AniLeo 
I can confirm this is a regression after #5038

**Comment by kd-11** [comment]
Duplicate of https://github.com/RPCS3/rpcs3/issues/5066
